common.tf

```diff
@@ -33,6 +33,8 @@ locals {
   enable_apiary_s3_log_management = var.apiary_log_bucket == "" ? true : false
   apiary_s3_logs_bucket           = local.enable_apiary_s3_log_management ? "${local.apiary_bucket_prefix}-s3-logs" : ""
   apiary_s3_hive_logs_bucket      = local.enable_apiary_s3_log_management ? "${local.apiary_s3_logs_bucket}-hive" : ""
+  apiary_system_bucket            = "${local.apiary_bucket_prefix}-${var.system_schema_name}"
```

Comment:
bucket names can't have underscores, and the default value of `var.system_schema_name` is `apiary_system`.  You need to use TF `replace` function to turn `_` to `-`.   Look at line 20 of this file for an example.